go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/containerd/containerd/v2
 
-go 1.24.3
+go 1.24.6
```

Comment:
If there are no significant improvements with this patch, I dont think we should be taking this is in which changes the minimum go version.